tests/e2e/cucumber/features/shares/share.feature

```diff
@@ -81,6 +81,10 @@ Feature: share
       | shareToBrian.md     | mdFile     | readme    |
       | shareToBrian.drawio | drawioFile |           |
       | sharedFile.txt      | txtFile    | some text |
+    And "Alice" edits the following resources
```

Comment:
is there any other feature file where we could cover up the tests for this edit seems odd here though.

Reply:
For the time being, I see no other suitable feature files to put the test for editing content of file. If later added, we can re-use this test.